.storybook/preview-body.html

```diff
@@ -3,30 +3,36 @@
 
 <style>
   /* Custom heading styles */
-  .sbdocs h1.storybook-heading:not(.innerZoomElementWrapper h1) {
+  .sbdocs h1:not(.chakra-heading.innerZoomElementWrapper h1) {
```

Comment:
The `not` statement needs to be updated.  These styles are now impacting the styles of the DS `Heading` components used on the story pages.  Take a look at the `Heading` component story page.